### PR TITLE
add Tide of Fortune (1993)

### PR DESCRIPTION
**Pull request by Nikuunj**
- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [ ] and will follow [the desired format][format-url]
- [ ] used image links that resolve correctly, are appropriately sized, and are served over HTTPS
- [x] Enhanced documentation

# Changes proposed in this pull request

> ### Project name : #319

> ### Description
* add Tide of Fortune (1993) , and add a quiz game

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -462,6 +462,31 @@
 | ------: | -------: | -----: |
 |   1 - 100 |        10 | 30-45m |
 
+### [Tide of Fortune (1993)](https://boardgamegeek.com/boardgame/3354/tide-fortune)
```
Comment (edm00se) [inline readme.md:465]: The title does not require the year.

File: readme.md [hunk 2]
```diff
@@ -462,6 +462,31 @@
...
+> The heart of the system is the Operations phase where players alternate activating formations (divisions) which execute Orders (Assault, Advance, Maneuver etc.).
+>
+> Players bid for Initiative and there are possibilities for the Reaction player to foil the initiative player's moves by having units react and do spoiling attacks.
+>
+> Losses can be applied to the HQ of a formation, not only directly to units.
+>
+> Low counter density, high maneuver. Medium-high complexity.
+
+
+![Tide of Fortune (1993)](https://cf.geekdo-images.com/vkDLqn-CMvTelKH0BvBBRA__imagepage/img/k2e0JHeSt5AwvNJco2rzFAXXaKM=/fit-in/900x600/filters:no_upscale():strip_icc()/pic3446255.jpg)
```
Comment (edm00se) [inline readme.md:484]: The image should be of ~250px in whichever is larger, height or width.

**Review by edm00se — CHANGES_REQUESTED** [review]
Thank you for submitting a game contribution to this list. The contribution to the list to add Tide of Fortune looks pretty good with a couple exceptions I've noted. Once those are updated it will be in good shape to merge.

There are however additional changes that should not be included in this PR. Those are:

- the change to `.gitignore`
- the addition of a "Quiz" webpage, scripts, and styles
  - Quiz.css
  - Quiz.html
  - Quiz.js

Those additions will need to be removed from this PR.

**Comment by Nikuunj** [comment]
i updated

**Pull request closed without merging**